sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1.java

```diff
@@ -656,6 +806,16 @@ public void processElement(ProcessContext context) throws Exception {
                   || (currentBatch.getMoreResults() == NOT_FINISHED));
         }
       }
+
+      @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
```

Comment:
I don't like how many DoFn's in this class repeat the same display data. It seems unnecessary. Is there a reason for it? E.g., is it that we can't provide display data for the composite transform itself, only for sub-transforms?

In any case, these could definitely benefit from `options` itself being a HasDisplayData - you'd just include it then.